modules/swagger-codegen/src/main/resources/cpprest/api-source.mustache

```diff
@@ -53,7 +53,22 @@ pplx::task<{{#returnType}}{{{returnType}}}{{/returnType}}{{^returnType}}void{{/r
     utility::string_t responseHttpContentType;
     
     // use JSON if possible
-    if ( responseHttpContentTypes.size() == 0 || responseHttpContentTypes.find(U("application/json")) != responseHttpContentTypes.end() )
+    if ( responseHttpContentTypes.size() == 0 )
+    {
+        {{#vendorExtensions.x-codegen-response.isPrimitiveType}}
```

Comment:
Since the original code was not iterating over the responses array of nodes for picking the preferred "Accept" content type, primativeType is not available without rethinking the approach to the code generation.

NOTE: I am still ramping on mustache.